admin/taxonomy/class-taxonomy.php

```diff
@@ -113,6 +113,7 @@ public function admin_enqueue_scripts() {
 			wp_localize_script( WPSEO_Admin_Asset_Manager::PREFIX . 'term-scraper', 'wpseoAnalysisWorkerL10n', array(
 				'url'                     => $analysis_worker_location->get_url( $analysis_worker_location->get_asset(), WPSEO_Admin_Asset::TYPE_JS ),
 				'keywords_assessment_url' => $used_keywords_assessment_location->get_url( $used_keywords_assessment_location->get_asset(), WPSEO_Admin_Asset::TYPE_JS ),
+				'log_level'               => WPSEO_Utils::get_analysis_worker_log_level(),
```

Comment:
We can make this into an object / PHP array:
```
log_level: {
     analysis_worker: 'INFO',
     ...
}
```
So we can add log levels for other parts of the analysis later on.

Reply:
Good point! Because you put it in the next lane I'm going to leave this to when we have more logs though 😉